### PR TITLE
Contracts update

### PR DESCRIPTION
**Pull request by u-hubar**
- Splitted ServiceAgreementV1 into ServiceAgreementV1 + CommitProofManager contracts
- Changed events in the contracts to include necessary parameters for ot-node
- Removed profile existence check if Profile.sol (not necessary because of onlyAdmin)
- Fixed all epochLength gets in ServiceAgreementV1 (use epochLength from agreement itself instead of ParametersStorage)

### CODE REVIEW

**Review by zeroxbt — APPROVED** [review]

**Review by zeroxbt — COMMENTED** [review]
Looks good! Just a reminder to make sure to add exports for new contract abis.

File: contracts/CommitManagerV1.sol [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+        uint16 epoch,
+        uint72 indexed identityId,
+        uint40 score
+    );
+    event Logger(bool value, string message);
+
+    string private constant _NAME = "CommitManagerV1";
+    string private constant _VERSION = "1.0.0";
+
+    bool[4] public reqs = [false, false, false, false];
```
Comment (NZT48) [inline contracts/CommitManagerV1.sol:36]: If we have this reqs set to true on the network currently, does it make sense to have it also initially set to true now?

File: contracts/ProofManagerV1.sol [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+        uint8 hashFunctionId,
+        uint16 epoch,
+        uint72 indexed identityId
+    );
+    event Logger(bool value, string message);
+
+    string private constant _NAME = "ProofManagerV1";
+    string private constant _VERSION = "1.0.0";
+
+    bool[4] public reqs = [false, false, false, false];
```
Comment (NZT48) [inline contracts/ProofManagerV1.sol:35]: same question here

**Review by NZT48 — APPROVED** [review]

File: contracts/CommitManagerV1.sol (hunk 1 above)
Comment (djordjekovac) [reply contracts/CommitManagerV1.sol:36]: I think that this should be false by default, since by default we want to perform the check.

**Review by djordjekovac — APPROVED** [review]

**Review by djordjekovac — CHANGES_REQUESTED** [review]
Can you please resolve conflicts before merging

**Review by djordjekovac — APPROVED** [review]

**Pull request merged**